uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsenceType.php

```diff
@@ -189,15 +189,22 @@ private function getCategoryLabel($categoryId) {
 
   /**
    * Loads required page resources
+   *
+   * @param bool $addExtraResources
    */
-  private function loadResources() {
+  private function loadResources($addExtraResources) {
```

Comment:
`addExtraResources` may not be descriptive enough. May be name it `addResourcesForAddingOrEditing`? The name I proposed may not be the best one, so please feel free to propose yours.

Reply:
👍 Thanks @igorpavlov. @tunbola what do you suggest? I also had difficulty on what to name it without using a long name 🤦‍♂️